app/grandchallenge/retina_core/management/commands/setannotationpermissions.py

```diff
@@ -0,0 +1,117 @@
+from django.core.management.base import BaseCommand
+
+from config.settings import PERMISSION_TYPES
+from grandchallenge.annotations.models import (
+    MeasurementAnnotation,
+    BooleanClassificationAnnotation,
+    IntegerClassificationAnnotation,
+    PolygonAnnotationSet,
+    LandmarkAnnotationSet,
+    ETDRSGridAnnotation,
+    CoordinateListAnnotation,
+    SinglePolygonAnnotation,
+    SingleLandmarkAnnotation,
+)
+from django.contrib.auth.models import Group
+from django.conf import settings
+from guardian.shortcuts import assign_perm, remove_perm
+
+# Existing annotation (name, codename) as of annotations.0001_initial
+ANNOTATION_CODENAMES = (
+    (BooleanClassificationAnnotation, "booleanclassificationannotation"),
+    (CoordinateListAnnotation, "coordinatelistannotation"),
+    (IntegerClassificationAnnotation, "integerclassificationannotation"),
+    (LandmarkAnnotationSet, "landmarkannotationset"),
+    (MeasurementAnnotation, "measurementannotation"),
+    (PolygonAnnotationSet, "polygonannotationset"),
+    (SingleLandmarkAnnotation, "singlelandmarkannotation"),
+    (SinglePolygonAnnotation, "singlepolygonannotation"),
+    (ETDRSGridAnnotation, "etdrsgridannotation"),
+)
+
+WARNING_TEXT = (
+    "Only {} model level permissions {}. No annotation objects found."
+)
+SUCCESS_TEXT = (
+    "Done! {} model level permissions and {} object level permissions {}."
+)
+
+
+def change_retina_permissions(remove=False):
+    if remove:
+        change_permission_func = remove_perm
+    else:
+        change_permission_func = assign_perm
+
+    olp_count = 0  # Object level permissions assigned/removed
+    mlp_count = 0  # Model level permissions assigned/removed
+    retina_admin_group = Group.objects.get(
+        name=settings.RETINA_ADMINS_GROUP_NAME
+    )
+    for (annotation_model, annotation_codename) in ANNOTATION_CODENAMES:
+        # Change user level object permissions to owners of annotations
+        for annotation in annotation_model.objects.all():
```

Comment:
Do we really want to set the permissions on each row here? Or are table permissions ok?